lib/matplotlib/tests/test_ticker.py

```diff
@@ -1352,6 +1352,17 @@ def test_engformatter_usetex_useMathText():
         assert x_tick_label_text == ['$0$', '$500$', '$1$ k']
 
 
+def test_locale_comma():
+    import locale
+    currentLocale = locale.getlocale()
+    locale.setlocale(locale.LC_ALL, 'deu_deu')
```

Comment:
This test will fail with "unsupported locale setting" if the locale is unavailable; at least this should be caught, and it may be better to just monkeypatch locale.format_string instead?